frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -173,6 +173,37 @@ const getRouteLabelsForEditor = (
     : routeLabels;
 };
 
+const ReceiverInfoTip: React.FC<ReceiverInfoTipProps> = ({ type }) => {
+  let msg: string = '';
```

Comment:
How about dropping the `let` and the `break` statements:

```
case 'Default':
   return 'Your default receiver will automatically receive al.....'
case 'Critical':
   return 'The routing labels for this receiv....'
case 'Watchdog':
  return 'The Watchdog alert fires consta....'
```

Reply:
Nice, much cleaner -thanks